pages/content/amp-dev/documentation/guides-and-tutorials/contribute/contribute-documentation/index.md

```diff
@@ -0,0 +1,41 @@
+---
+$title: Contribute content 
+$order: 0
+description: 'Get started contributing content to amp.dev'
+formats:
+  - websites
+  - stories
+  - ads
+  - email
+author: CrystalOnScript
+---
+
+Documentation is the starting point for developers learn how to build successful websites, stories, ads, and dynamic emails with AMP. The core AMP content team is a small one, but our responsibility is big. We appreciate your help! Contributors can fix typos, correct out of date information, and write fresh content. Use this page to get started learning the ins and outs of becoming a content contributor. 
+
+Thanks for your interest in contributing to the AMP Project, welcome to the team! 
+
+# Get started
+
+amp.dev hosts our documentation and the AMP team collaborates on [GitHub](https://github.com/ampproject). Follow the amp.dev [repository’s README](https://github.com/ampproject/amp.dev) to run amp.dev on your local machine. This is essential for ensuring correct asset rendering and formatting. However, for simple fixes, such as typos, you can edit the file directly on GitHub.
```

Comment:
Good opportunity to link to the wg-outreach repo / working group (or maybe further up when talking about the core AMP content team)